app/views/groups/edit.html.slim

```diff
@@ -0,0 +1,4 @@
+.row
+  .col-lg-12 
+    h1= "Edit #{@group.name}"
```

Comment:
You can remove the quotes and say
```
h1= Edit #{@group.name}
```

Reference: https://github.com/slim-template/slim#text-interpolation